src/documentation/literal.rs

```diff
@@ -531,6 +531,72 @@ struct Five;
                 },
             },
         },
+        // 6
+        Triplet {
+            source: r#"
+
+    /// if a layer is provided a identiacla "input" and "output", it will only be supplied an
+    fn compute_in_place(&self) -> bool {
+        false
+    }
+
+"#,
+            extracted: r#"" if a layer is provided a identiacla \"input\" and \"output\", it will only be supplied an""#,
+            trimmed: r#" if a layer is provided a identiacla \"input\" and \"output\", it will only be supplied an"#,
```

Comment:
is this test related to this pr?

Reply:
When testing this PR on https://github.com/spearow/juice there were some additional bugs being uncovered, so I added the testcase here, a few more (unrelated) fixes pending commitment to this branch